build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+
+if [ $(git branch | grep \* | cut -d ' ' -f2) = "master" ];then
+    echo "For nown, only build the branches"   
+    return
+fi
+
+echo "Building branch now ..."
+
+if [ ! -d libs-sdks ];then
+    git clone https://github.com/martinpaljak/oracle_javacard_sdks.git libs-sdks
+fi
+
+mkdir -p src/main/java/org/idpass
+
+t=tmp$$
+mkdir -p $t
+git clone git@github.com:idpass/card-tools-applet.git $t/tools
+git clone git@github.com:idpass/card-auth-applet.git $t/auth
+git clone git@github.com:idpass/card-sam-applet.git $t/sam
+
+ln -sf `pwd`/$t/tools/src/org/idpass/tools `pwd`/src/main/java/org/idpass/
+ln -sf `pwd`/$t/auth/src/org/idpass/auth `pwd`/src/main/java/org/idpass/
+ln -sf `pwd`/$t/sam/src/org/idpass/sam `pwd`/src/main/java/org/idpass/
```

Comment:
Could we use the git submodule feature instead to pull the repos at the expected version?

Could we change the structure of the repo(s) so we do not have to do those changes of paths? It will make development easier.

Reply:
I've reconfigured `.gitmodules` now to checkout to common `src/main/java/org/idpass/` in my latest commit to the master branch. However, each repo still has nested inside it's own `src/org/idpass/` hieararchy. A gradle setting now comes into mind, and will search for it..

Reply:
A new PR is submitted which is a branch from a correctly re-initialized master with respect to submodule references.